README.md

```diff
@@ -41,6 +34,22 @@ Several other validation scripts and tools with more flexibility such as validat
 bin/validate-datapackage [-v remote_spec_ref | -l local_spec_path] [-d dataset_path]
 ```
 
+### Specific files
+
+Specific files can be validated by running the frictionless framework against them and their corresponding schemas as follows:
+
+```sh
+frictionless validate vehicles.csv --schema https://raw.githubusercontent.com/TIDES-transit/TIDES/main/spec/schema.vehicles.json
```

Comment:
This requires `--schema-sync`, or optional fields that are left out will cause validation errors